client/packages/common/src/ui/components/inputs/Filters/FilterMenu.tsx

```diff
@@ -99,14 +126,34 @@ const FilterMenuItem: FC<{ onClick: () => void; label: string }> = ({
 );
 
 const getFilterOptions = (
-  filters: FilterDefinition[],
+  filters: (FilterDefinition | GroupFilterDefinition)[],
   activeFilters: FilterDefinition[]
 ) => {
   const activeFilterCodes = activeFilters.map(fil => fil.urlParameter);
 
   return filters
-    .filter(fil => !activeFilterCodes.includes(fil.urlParameter))
-    .map(fil => ({ label: fil.name, value: fil.urlParameter }));
+    .filter(fil =>
+      fil.type === 'group'
+        ? !fil.elements.every(flatFil =>
+            activeFilterCodes.includes(flatFil.urlParameter)
+          )
+        : !activeFilterCodes.includes(fil.urlParameter)
+    )
```

Comment:
Even though we only display the "Group" name in the menu, we still need to check the inner elements against the activeFilter list to determine whether or not to include the group in the options list.